peer/pendingheap/list_test.go

```diff
@@ -149,13 +149,13 @@ func TestPeerHeapList(t *testing.T) {
 			peerListActions: []PeerListAction{
 				StartAction{},
 				UpdateAction{AddedPeerIDs: []string{"1", "2", "3", "4", "5", "6"}},
-				ChooseAction{ExpectedPeer: "1"},
-				ChooseAction{ExpectedPeer: "2"},
+				ChooseAction{ExpectedPeer: "6"},
```

Comment:
Would it be possible to disable shuffling for this test? The shuffle eliminates some clarity that this test verifies that the fewest-pending-requests peer selector degenerates to round-robin when load is perfectly even.